src/anyio_marketstore/results.py

```diff
@@ -1,7 +1,7 @@
 from typing import List, Dict
 
 import numpy as np
-import pandas as pd
+# import pandas as pd
```

Comment:
Yeah just leaving this commented for now so we have it for reference if anyone ends up wanting it later.